Server/src/tests/controllers/CategoryController.spec.ts

```diff
@@ -33,98 +40,95 @@ describe('Category Controller', () => {
     mockRequest = {
       body: validUploadData
     }
-    await controller.createCategory(mockRequest as Request, mockResponse as Response)
+    await createCategory(mockRequest as Request, mockResponse as Response, 201)

```

Comment:
same here, createCategory only implies that a category will be created, doesnt imply also there will be expect statements